webpack.config.js

```diff
@@ -0,0 +1,33 @@
+const path = require('path')
+
+module.exports = (_, argv) => ({
+  entry: path.normalize(`${__dirname}/src/renderer/main.js`),
+  target: 'electron-main',
+  output: {
+    path: path.normalize(`${__dirname}/static`),
+    filename: 'bundle.js',
```

Comment:
`static/bundle.js` :+1:

Reply:
are you saying to change `bundle.js` to `static/bundle.js`

Reply:
seems to work, the `path` seems to be properly taking care of the directory

Reply:
No, I'm saying it was a good thing to do.